semgrep-core/src/core/il/IL.ml

```diff
@@ -351,10 +356,15 @@ and node_kind =
   | NTodo of stmt
 [@@deriving show { with_path = false }]
 
+and augmented_node_kind =
```

Comment:
What do we gain by having an `augmented_node_kind`? Why not just adding `Func` to `node_kind`?